CMakeLists.txt

```diff
@@ -135,6 +136,103 @@ if (NOT (Boost_THREAD_LIBRARY STREQUAL monero_Boost_THREAD_LIBRARY_RELEASE))
   message(FATAL_ERROR "Boost libraries for monero build differs from this project")
 endif()
 
+# hidapi
+# https://github.com/moneroexamples/openmonero/blob/18370af8fa51a0e9d05dfe678dcd5bcb7fad30f7/cmake/FindHIDAPI.cmake#L26-L60
```

Comment:
HIDAPI changes in this file are here to fix compiler errors like:

```
/usr/bin/ld: /monero/build/release/src/device/libdevice.a(device_io_hid.cpp.o): in function `hw::io::device_io_hid::disconnect() [clone .localalias]':
device_io_hid.cpp:(.text+0x32): undefined reference to `hid_close'
```

Pretty much just copy pasted the `openmonero` code linked above.